lib/src/Pos/Launcher/Resource.hs

```diff
@@ -444,7 +444,7 @@ createTransportTCP addrInfo = do
         Left e -> do
             logError $ sformat ("Error creating TCP transport: " % shown) e
             throwM e
-        Right transport -> return (concrete transport, liftIO $ NT.closeTransport transport)
+        Right transport -> return (concrete transport, pass {-# liftIO $ NT.closeTransport transport #-})
```

Comment:
kommentiki

Reply:
Why is this `{-#` instead of simply `{-`? It's not a pragma...

Reply:
Oh, I forgot the syntax :unamused: 
I should also leave issue ID and some explanation here I suppose.